src/components/wallet/TurnOffBlockchain/TurnOffBlockchain.js

```diff
@@ -0,0 +1,108 @@
+/**
+ * Copyright 2017–2018, LaborX PTY
+ * Licensed under the AGPL Version 3 license.
+ */
+
```

Comment:
Th file uses JSX syntax but has `js` extension. I think the extension should be jsx